### PR TITLE
checkpoint: fix handling of R81 fields

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

Fixes handling of numeric and duplicated fields and removes undocumented fields that appear to be new in R81.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Updates elastic/beats#32380

### CODE REVIEW

**Review by r00tu53r — APPROVED** [review]
LGTM!

File: packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -471,6 +471,10 @@ processors:
       field: checkpoint.ifdir
       target_field: network.direction
       ignore_missing: true
+  - convert:
+      field: checkpoint.bytes
+      type: long
+      ignore_missing: true
   - rename:
       field: checkpoint.bytes
       target_field: network.bytes
```
Comment (r00tu53r) [inline packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml:480]: Thinking of doing this in a single step - 
```suggestion
  - convert:
      field: checkpoint.bytes
      target_field: network.bytes
      type: long
      ignore_missing: true
```

File: packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -508,6 +512,10 @@ processors:
         } else if (iana_number == '132') {
             ctx.network.transport = 'sctp';
         }
+  - convert:
+      field: checkpoint.packets
+      type: long
+      ignore_missing: true
   - rename:
       field: checkpoint.packets
       target_field: network.packets
```
Comment (r00tu53r) [inline packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml:521]: Thinking of doing this in a single step - 
```suggestion
  - convert:
      field: checkpoint.packets
      type: long
      target_field: network.packets
      ignore_missing: true
```

File: packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (efd6) [reply packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml:480]: That leaves the value in `checkpoint.bytes`.

File: packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml (hunk 2 above)
Comment (efd6) [reply packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml:521]: as above

**Pull request merged**